src/components/pill/pill.component.tsx

```diff
@@ -49,31 +55,48 @@ export const Pill = ({
   size = "M",
   ariaLabelOfRemoveButton = "remove pill",
   ...rest
-}: PillProps) => (
-  <StyledPill
-    inFill={fill}
-    colorVariant={colorVariant}
-    isDeletable={!!onDelete}
-    pillRole={pillRole}
-    size={size}
-    borderColor={borderColor}
-    onClick={onClick}
-    {...tagComponent("pill", rest)}
-    maxWidth={maxWidth}
-    wrapText={wrapText}
-    {...rest}
-  >
-    {children}
-    {onDelete && (
-      <IconButton
-        onClick={onDelete}
-        data-element="close"
-        aria-label={ariaLabelOfRemoveButton}
-      >
-        <Icon type="cross" />
-      </IconButton>
-    )}
-  </StyledPill>
-);
+}: PillProps) => {
+  if (
```

Comment:
praise: good catch for ensuring consumer only uses this variant on a dark background